### PR TITLE
Add More Practical Features

### PR DESCRIPTION
**Pull request by askdkc**
- NOT search function added
- Google-Like Search feature added

### CODE REVIEW

File: how-to/postgrest.md [hunk 1]
```diff
@@ -293,3 +306,95 @@ BEGIN
...
+      (
+        "unify_kana", true,
+        "unify_to_romaji", true,
+        "unify_hyphen_and_prolonged_sound_mark", true
+      )',
+    tokenizer = 'TokenNgram("unify_symbol", false, "unify_alphabet", false, "unify_digit", false)'
+  );
+```
+
+### Creating a Stored Function for Keyword-Based Search:
```
Comment (kou) [inline how-to/postgrest.md:360]:
```suggestion
### Creating a Stored Function for Keyword-Based Search
```

**Comment by kou** [comment]
Thanks!

**Comment by askdkc** [comment]
This one merged so quickly 👀

**Comment by askdkc** [comment]
I will translate these after `po` is created👍

**Comment by kou** [comment]
Thanks but please wait for it until I prepared Transifex integration.

**Pull request merged**